modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -23,6 +23,9 @@ var (
 	contractClientState    = []byte{1}
 	contractConsensusState = []byte{2}
 	ErrMockContract        = errors.New("mock contract error")
+	MockClientStateBz      = []byte("client-state-data")
+	MockValidProofBytes    = []byte("valid proof")
+	MockInValidProofBytes  = []byte("invalid proof")
```

Comment:
Just a nit, but maybe for consistency we could choose either `Bz` or `Bytes`.

Reply:
also super nit, `MockInvalidProofBytes` or `MockInvalidProofBz` instead of `InValid` 🙈

Reply:
My bad, my english is good I swear 😭